gdx/src/com/badlogic/gdx/graphics/g2d/PixmapPacker.java

```diff
@@ -101,6 +101,7 @@
 	Format pageFormat;
 	int padding;
 	boolean duplicateBorder;
+	Color currentTransparentColor = new Color(0f, 0f, 0f, 0f);

```

Comment:
Not sure I agree with your reasoning. This is not really any different than SpriteBatch's getColor/setColor.